.github/workflows/ci.yml

```diff
@@ -4,15 +4,20 @@ on:
     - cron: "0 0 * * *"
   push:
     branches:
-      - master
+      - main
```

Comment:
To confirm, is the `demo` plugin's default branch changing from `master` to `main`? (I'm not sure it's worth doing this unless we were going to be consistent everywhere.)

Reply:
This is caused by mage taking care of the common files (aka. ci pipelines). The ci files are present [here](https://github.com/mattermost/mattermost/tree/fe126ed2356eb8215556c525ebc080f77310b239/server/public/pluginmage/assets) and the tooling updates it if there's a different with the file in the plugin repository.